test/test_list.rb

```diff
@@ -222,6 +222,24 @@ def setup
   end
 end
 
+class AddAtTopTest < ActsAsListTestCase
+  def setup
+    setup_db(unique: true)
+  end
+
+  def test_insert
+    new = ArrayScopeListMixin.create!
+    assert_equal 1, new.pos
+    new = ArrayScopeListMixin.create!
+    assert_equal 2, new.pos
+    new = ArrayScopeListMixin.create!
+    assert_equal 3, new.pos
+    new = ArrayScopeListMixin.create!(pos: 1)
+    assert_equal 1, new.pos
```

Comment:
Seems like this test fails with sqlite even without my change above. I've no idea right now why or how this could be fixed.